admin_solutions/authentication.adoc

```diff
@@ -14,9 +14,236 @@ toc::[]
+to store the configuration file created by the start command:
++
+----
+$ openshift start master --public-master=<apiserver> --write-config=<directory>
+----
++
+For example:
++
+----
+$ openshift start master --public-master=https://myapiserver.com:8443 --write-config=ldapconfig
+----
++
+[NOTE]
+====
+If you are installing with Ansible, then you must add the
+`*identityProvider*` configuration to the Ansible playbook.
+If you use the following steps to modify your configuration manually after installing with Ansible, then you will lose any modifications whenever you re-run the install tool or upgrade.
+====
++
+. Edit the new *_master-config.yaml_* file's `*identityProviders*` stanza.
+. Copy
+link:../install_config/configuring_authentication.html#ldap-example-config[the example `*LDAPPasswordIdentityProvider*` configuration]
+and paste it to replace the existing stanza.
+. Make the following modifications to the `*identityProviders*` stanza:
+.. Change the provider `*name*` ("my_ldap_provider") to something unique and
+relevant to your deployment. This name is prefixed to the returned user name to
+form an identity name.
+.. If required,
+link:../install_config/configuring_authentication.html#mapping-identities-to-users[change `*mappingMethod*`] to control how mappings are established between the
+provider's identities and user objects.
+.. Change `*id*` to the attribute to use as the identity.
+This option can accept multiple attributes. If more than one is specified, they will be checked in order and the first non-empty attribute will be used.
+At least one
+attribute is required. If none of the listed attribute have a value, then
+authentication fails.
+.. Change `*email*` to the attribute to use as the email address.  This option
+can accept multiple attributes. If more than one is specified, they will be
+checked in order and the first non-empty attribute will be used.
+.. Change `*name*` to the attribute to use as the display name.  This option can
+accept multiple attributes. If more than one is specified, they will be checked
+in order and the first non-empty attribute will be used.
+.. Change `*preferredUsername*` to the attribute to use as the preferred user
+name when provisioning a user for this identity. This option can accept multiple
+attributes. If more than one is specified, they will be checked in order and the
+first non-empty attribute will be used.
+.. Change the `*ca*` to the certificate bundle to use in order to validate
+server certificates for the configured URL. If empty, system trusted roots are
+used. This setting only applies if *insecure: false*. If the LDAP server
+requires a different certificate chain, this attribute should contain the
+filesystem path of that certificate or certificate bundle.
+.. If required, modify the `*insecure*` parameter. The default is *false*. When *true*, no TLS connection is made to the server. When *false*, `ldaps://` URLs connect using TLS, and `ldap://` URLs are upgraded to TLS.
```

Comment:
Thanks.

OK, then it should probably be noted that the insecure parameter must be "false" when using `ldaps://` URIs.